test/model/message_list_test.dart

```diff
@@ -1703,6 +1703,25 @@ void main() {
         }
       }
     });
+    group('topics compared case-insensitively', () {
```

Comment:
bump https://github.com/zulip/zulip-flutter/pull/1106#discussion_r1889259739

Reply:
Oh, sorry for overlooking that. Done now!